genienlp/data_utils/example.py

```diff
@@ -172,6 +172,33 @@ def from_examples(examples: Iterable[Example], numericalizer):
             tokenized_answers = numericalizer.process_classification_labels(
                 all_context_plus_questions, [ex.answer for ex in examples]
             )
+        elif getattr(examples, 'is_sequence_classification', False):
+            # align labels
+            answers = [
+                [
+                    int(ex.answer),
+                ]
+                for ex in examples
+            ]
+
+            batch_decoder_numerical = []
+            if numericalizer.decoder_vocab:
+                for i in range(len(answers)):
+                    batch_decoder_numerical.append(numericalizer.decoder_vocab.encode(answers[i]))
+            else:
+                batch_decoder_numerical = [[]] * len(answers)
```

Comment:
batch_decoder_numerical doesn't make much sense for a sequence_classification task, it is meant for seq2seq models that use a different output vocabulary.